### ISSUE TITLE
Allow story iframes to be opened in their own tab

### ISSUE THREAD

**Issue by peey** [storybookjs/storybook#13843]
**Is your feature request related to a problem? Please describe**
Devtools don't often work on iframes.

Devtools (like Angular's Augury) rely on detecting if the page has a global instance of the framework running (and then hooking into it through js monkeypatching and other brittle techniques). 

Since storybook stories each run in their own iframe, devtools have to be aware of this case and specifically add support for it (increasing development burden and often taking several months / years for support to arrive).

**Describe the solution you'd like**

The other option is to use browser's built-in "open frame in new tab" mechanism to open the component code without any interfering storybook code. 

Currently trying "open frame in new tab" just redirects back to storybook main instance.

Then devtools could be used out of the box in that tab.

**Describe alternatives you've considered**

Alternative would be to ask *each* devtools developer to support storybook as a special case, which as I described is a bad option. Tools have enough problems as is (e.g. keeping up with support for latest framework versions). This will cause such feature requests to be mostly neglected.

**Are you able to assist to bring the feature to reality?**

No. I'm not familiar with devtools or storybook internals. Only insight I have into this problem is that if stories can be opened in their own tabs, then everything should work without having to write any extra code. 

**Additional context**

Please ask if anything's required to better understand this request.

**Comment by peey** [comment]
This is already supported ("open canvas in new tab"). I just did not find the right control for it in storybook UI.

Question: why does the browser's "open frame in new tab" not work? Should I turn this into a bug report instead?

**Comment by shilman** [comment]
Which version are you using? I've verified that the latest stable and alpha versions both do the right thing (at least for statically built storybooks -- have not checked in dev mode)

See:
- [latest](https://master--storybookjs.netlify.app/official-storybook/?path=/story/addons-a11y-basebutton--default)
- [next](https://next--storybookjs.netlify.app/official-storybook/?path=/story/addons-a11y-basebutton--default)

**Comment by peey** [comment]
You're right, the netlify versions you linked above seem to be working with right click, but mine isn't.

I initialized the project using `npx sb init` a few days ago, these are the versions in my `package.json`: 
```
    "@storybook/addon-actions": "^6.1.17",
    "@storybook/addon-essentials": "^6.1.17",
    "@storybook/addon-links": "^6.1.17",
    "@storybook/angular": "^6.1.17",
```

I think I'm using dev mode (using `npm run storybook`, I don't know about static mode)

**Comment by shilman** [comment]
Do you have a repro repo you can share?

**Comment by peey** [comment]
Nope. I can try initializing a fresh angular demo project and then storybook in it later this week. 

Might this be a version-related issue (your netlify demo is using the latest version, my angular project using outdated version)?
